src/chrome/content/rules/Heroku.xml

```diff
@@ -7,15 +7,16 @@
 <ruleset name="Heroku">
 
 	<target host="*.herokuapp.com" />
-
-	<exclusion pattern="^http://mc\.heroku\.com/"/>
+		<test url="http://blog.herokuapp.com/" />
+		<test url="http://chempro101.herokuapp.com/" />
+		<test url="http://cosio.herokuapp.com/" />
+		<test url="http://voices.herokuapp.com/" />
 
 	<!--	Avoiding cross-domain cookies.
 						-->
 	<securecookie host="^[\w\-]+\.herokuapp\.com$" name=".*" />
 
-
-	<rule from="^http://([^/:@\.]+\.)?heroku(app)?\.com/"
-		to="https://$1heroku$2.com/" />
+	<rule from="^http://([^/:@\.]+\.)?herokuapp\.com/"
```

Comment:
Can you please avoid this complex regex?

Reply:
@J0WI, this will cause invalid cert warnings on `www.*.herokuapp.com` but it's just a page saying "There's nothing there". Does it matter?

Reply:
Or did you just mean 

```xml
    <rule from="^http://([\w-]+)\.herokuapp\.com/"
        to="https://$1.herokuapp.com" />
```

?